### ISSUE TITLE
[STTools] Cannot handle IceAct OMKey(0,1) (Trac #1713)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1705]
STTools breaks L2 processing because of IceAct geometry.

Fix required for season 2015 and 2016

Migrated from https://code.icecube.wisc.edu/ticket/1713
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:58", 
    "description": "STTools breaks L2 processing because of IceAct geometry.\n\nFix required for season 2015 and 2016", 
    "reporter": "joertlin", 
    "cc": "mwolf", 
    "resolution": "fixed", 
    "_ts": "1550067178841456", 
    "component": "combo reconstruction", 
    "summary": "[STTools] Cannot handle IceAct OMKey(0,1)", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-05-19T15:05:04", 
    "milestone": "", 
    "owner": "joertlin", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160519T15:05:58: joertlin changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160519T17:33:16: 
* joertlin commented:

> In changeset 146131:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="146131"
> Added IceAct OMKey(0,1). Closes #1705
> ```


* joertlin changed resolution from "" to "fixed"
* joertlin changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:58: 
* olivas changed milestone from "May 2016" to ""
* olivas commented:

> Milestone May 2016 deleted
